back-end/src/main/java/nl/hu/serious_game/SeedLevelTemplateRunner.java

```diff
@@ -41,7 +50,7 @@ public LevelTemplate createLevel1() {
         LevelHouse house2 = new LevelHouse(dayProfile, new HouseOptions());
```

Comment:
De houses in deze runner moeten ook maxSolarPanels en max Batteries krijgen, lijkt me - de constructor zonder die parameters kan mss ook weg